functests/performance/updating_profile.go

```diff
@@ -113,12 +118,55 @@ var _ = Describe("[rfe_id:28761] Updating parameters in performance profile", fu
 			}
 		}
 	})
+
+	It("[test_id:28440]Verifies that nodeSelector can be updated in performance profile", func() {
+		var newWorkerNode corev1.Node
```

Comment:
What about using a pointer?
Makes the check if you found a node a bit nicer later on (skip if node is nil, and not if node.name = "")

Reply:
done, can you please take a look on changes